src/components/menu-left/MenuLeft.vue

```diff
@@ -1,19 +1,155 @@
 <template>
-  <div class="column is-narrow menu-left">
-    <h3>Menu left</h3>
+  <div class="menu-left">
+    <b-field
+      :label="`Beat: ${beat} / ${selectedSSBeats}`"
+      data-test="menu-left--beat"
+    >
+      <b-slider
+        v-model="beat"
+        class="beat-slider"
+        data-test="menu-left--beat-slider"
+        :min="1"
+        :max="selectedSSBeats"
+        :tooltip="false"
+      />
+    </b-field>
+    <div class="buttons beat-buttons">
+      <b-button
+        type="is-primary"
+        icon-left="arrow-left-bold"
+        data-test="menu-left--decrement-beat"
+        @click="decrementBeat"
+      />
+      <b-button
+        type="is-primary"
+        icon-left="arrow-right-bold"
+        data-test="menu-left--increment-beat"
+        @click="incrementBeat"
+      />
+    </div>
+    <p class="label">Stuntsheets</p>
+    <b-menu>
+      <b-menu-list>
+        <b-menu-item
+          v-for="(stuntSheet, index) in stuntSheets"
+          :key="stuntSheet.title"
+          class="stuntsheet"
+          :active="selectedSS === index"
+          data-test="menu-left--ss"
+          @click="selectedSS = index"
+        >
+          <template slot="label">
+            {{ `${index + 1}) ${stuntSheet.title}` }}
+            <b-icon
+              class="stuntsheet-edit is-pulled-right"
+              icon="pencil"
+              size="is-small"
+              @click.native="stuntSheetModalActive = true"
+            />
+          </template>
+        </b-menu-item>
+      </b-menu-list>
+    </b-menu>
+    <b-button
+      type="is-text"
+      size="is-small"
+      expanded
+      data-test="menu-left--add-ss"
+      @click="addStuntSheet"
+    >
+      Add Stuntsheet
+    </b-button>
+    <b-modal
+      :active.sync="stuntSheetModalActive"
+      has-modal-card
+      trap-focus
+      data-test="menu-left--ss-modal"
+    >
```

Comment:
Should this also show up for a new stuntsheet when adding? (pressing "add stuntsheet" adds the stuntsheet and brings up the stuntSheetModal for that new sheet)